presto-docs/src/main/sphinx/release/release-0.153.rst

```diff
@@ -6,10 +6,72 @@ General Changes
 ---------------
 
 * Fix incorrect results for grouping sets when ``task.concurrency`` is greater than one.
+* Fix silent numeric overflow when casting ``INTEGER`` to large ``DECIMAL``
+  types.
+* Fix issue where GROUP BY () would produce no results if the input
+  had no rows.
+* Fix null handling in :func:`array_distinct` where applied to ``array(bigint)`` type.
+* Fix potential thread deadlock in coordinator.
+* Fix rare correctness issue with an aggregation on a single threaded `RIGHT JOIN` when
+  `task.concurrencty` is `1`.
+* Fix query failure when casting a map with null elements in values.
+* Fix failure when view column names contain upper-case letters.
+* Fix potential performance regression due to skew issue when
+  grouping or joining on columns of the following types: ``TINYINT``,
+  ``SMALLINT``, ``INTEGER``, ``BIGINT``, ``REAL``, ``DOUBLE``,
+  ``COLOR``, ``DATE``, ``INTERVAL``, ``TIME``, ``TIMESTAMP``.
+* Fix potential memory leak for delete queries.
 * Make resource group configuration more flexible. See "SPI Changes" below, and the
   :doc:`resource groups documentation </admin/resource-groups>`.
 * Improve performance of :ref:`array_type` when underlying data is dictionary encoded.
 * Improve performance of ORC reader when decoding dictionary encoded :ref:`map_type`.
+* Allow running Presto with early-access Java versions.
+* Allow subqueries in non-equality outer join criteria.
+* Add initial support for correlated subqueries.
+* Add support for prepared statements.
+* Add :func:`type_of` for discovering expression types.
+* Add ``DOUBLE PRECISION`` as an alias to ``DOUBLE`` type.
+* Add decimal support to :func:`avg`, :func:`ceil`, :func:`floor`, :func:`round`,
+  :func:`truncate`, :func:`abs`, :func:`mod`, :func:`sign`.
+* Add :func:`shuffle` function.
+* Add :func:`empty_approx_set` function.
+* Compile `OUTER JOIN` with filer function.
```

Comment:
What does "filer function" mean? Is that a typo? Explain in terms that an end user can understand.